lcli/src/main.rs

```diff
@@ -281,6 +256,109 @@ fn main() {
                         .help("The value for state.genesis_time."),
                 )
         )
+        .subcommand(
+            SubCommand::with_name("new-testnet")
+                .about(
+                    "Produce a new testnet directory.",
+                )
+                .arg(
+                    Arg::with_name("testnet-dir")
+                        .long("testnet-dir")
+                        .value_name("DIRECTORY")
+                        .takes_value(true)
+                        .help("The output path for the new testnet directory. Defaults to ~/.lighthouse/testnet"),
+                )
+                .arg(
+                    Arg::with_name("min-genesis-time")
+                        .long("min-genesis-time")
+                        .value_name("UNIX_SECONDS")
+                        .takes_value(true)
+                        .help("The minimum permitted genesis time. For non-eth1 testnets will be
+                              the genesis time. Defaults to now."),
+                )
+                .arg(
+                    Arg::with_name("min-genesis-active-validator-count")
+                        .long("min-genesis-active-validator-count")
+                        .value_name("INTEGER")
+                        .takes_value(true)
+                        .default_value("16384")
+                        .help("The number of validators required to trigger eth2 genesis."),
+                )
+                .arg(
+                    Arg::with_name("min-genesis-delay")
+                        .long("min-genesis-delay")
+                        .value_name("UNIX_SECONDS")
```

Comment:
I think this param could be called `SECONDS` instead of `UNIX_SECONDS`, as it has nothing to do with the UNIX epoch